nixos/lib/make-disk-image.nix

```diff
@@ -17,7 +17,7 @@
   # either "efi" or "hybrid"
   # This will be undersized slightly, as this is actually the offset of
   # the end of the partition. Generally it will be 1MiB smaller.
-  bootSize ? "256MiB"
+  bootSize ? "256M"
```

Comment:
In https://github.com/NixOS/nixpkgs/pull/203641/commits/2900974e7604b57ff332e2401f7f32cde8303611 you are not just changing documentation but also revert `

```diff
- bootSize ? "256MiB"
+ bootSize ? "256M"
```

This should be in https://github.com/NixOS/nixpkgs/pull/203641/commits/5380f26a8b806b3ac80769b3e3094812e76e97bb

Reply:
Right